docs/_tutorials/convert-existing-site-to-jekyll.md

```diff
@@ -468,50 +476,47 @@ search: exclude
 </urlset>{% endraw %}
 ```
 
-Again, we're using a for loop here to iterate through all posts and pages to add them to the sitemap.
+Again, we're using a `for` loop here to iterate through all posts and pages to add them to the sitemap.
 
 You can also auto-generate your sitemap by adding a gem called [`jekyll-sitemap`][jekyll-sitemap]. This gem will also work on GitHub Pages.
 
-
 ## 12. Add external services
 
 For other services you might need (such as contact forms, search, comments, and more), look for third-party services. For example, you might use the following:
 
-* For comments: [Disqus](https://disqus.com/)
-* For a newsletter: [Tinyletter](https://tinyletter.com/)
-* For contact forms: [Wufoo](https://www.wufoo.com/)
-* For search: [Algolia Docsearch](https://community.algolia.com/docsearch/)
+*   For comments: [Disqus](https://disqus.com/)
+*   For a newsletter: [Tinyletter](https://tinyletter.com/)
+*   For contact forms: [Wufoo](https://www.wufoo.com/)
+*   For search: [Algolia Docsearch](https://community.algolia.com/docsearch/)
 
 For more details on services for static sites, see the [Third Parties](https://learn.cloudcannon.com/jekyll-third-parties/) list and tutorials from CloudCannon.
 
 Your Jekyll pages consist of HTML, CSS, and JavaScript, so pretty much any code you need to embed will work without a problem.
 
 As you integrate code for these services, note that **if a page in your Jekyll site doesn't have front matter tags, Jekyll won't process any of the content in that page.** The page will just be passed to the `_site` folder when you build your site.
 
-If you do want Jekyll to process some page content (for example, to populate a variable that you define in your site's config file), just add front matter tags to the page. If you don't want any layout applied to the page, specify `layout: null` like this:
+If you do want Jekyll to process some page content (for example, to populate a variable that you define in your site's config file), just add front matter tags to the page. If you don't want any layout applied to the page, specify `layout: nil` like this:
```

Comment:
@DirtyF its `layout: null` instead of `layout: nil`

ref: [Ben's plugin](https://github.com/benbalter/jekyll-default-layout#disabling)
/cc @parkr @benbalter